src/main/java/lotto/domain/lotto/exception/InvalidLottoNumberException.java

```diff
@@ -0,0 +1,10 @@
+package lotto.domain.lotto.exception;
+
+public class InvalidLottoNumberException extends RuntimeException {
+
+    private static final String message = "유효하지 않은 로또 번호들 입니다.";
```

Comment:
```suggestion
    private static final String MESSAGE = "유효하지 않은 로또 번호들 입니다.";
```

상수는 대문자로 작성해주는 것이 관례입니다 :)